u2f-gae-demo/src/com/google/u2f/gaedemo/storage/TokenStorageData.java

```diff
@@ -119,8 +113,7 @@ public boolean equals(Object obj) {
     TokenStorageData that = (TokenStorageData) obj;
     return (this.enrollmentTime == that.enrollmentTime)
         && SecurityKeyData.containSameTransports(this.transports, that.transports)
-        && (this.counter == that.counter)
-        && Arrays.equals(this.keyHandle, that.keyHandle)
+        && (this.counter == that.counter) && Arrays.equals(this.keyHandle, that.keyHandle)
```

Comment:
Not positive this is better. It's true that these fit on one line. On the other hand, leaving it as it was allows you to scan each predicate one line at a time.